RecoHGCal/TICL/plugins/TrackstersProducer.cc

```diff
@@ -92,9 +90,17 @@ TrackstersProducer::TrackstersProducer(const edm::ParameterSet& ps, const Tracks
           consumes<std::vector<TICLSeedingRegion>>(ps.getParameter<edm::InputTag>("seeding_regions"))),
       itername_(ps.getParameter<std::string>("itername")) {
   if (doNose_) {
+    auto plugin = ps.getParameter<std::string>("patternRecognitionBy");
```

Comment:
This and next line can be moved out of the `if`, removing their repetiion in the `else`.